lib/ui/view/forgot_password_otp/forgot_password_otpviewmodel.dart

```diff
@@ -17,4 +17,4 @@ class ForgotPasswordOtpViewModel extends BaseViewModel {
   final otpController = TextEditingController();
```

Comment:
This would add a material import to your viewmodel, please remove... Use [stacked FormView](https://www.filledstacks.com/post/building-flutter-login-and-sign-up-forms/#form-functionality)